pandas/_libs/algos_rank_helper.pxi.in

```diff
@@ -44,6 +44,10 @@ def rank_1d_{{dtype}}(object in_arr, ties_method='average', ascending=True,
 
         {{if dtype == 'object'}}
         ndarray sorted_data, values
```

Comment:
you can put the sorted_nanmask for all dtypes

Reply:
Yes , you are right. I should add for all dtypes.